Formula/sonar-scanner.rb

```diff
@@ -0,0 +1,32 @@
+class SonarScanner < Formula
+  desc "Launcher to analyze a project with SonarQube"
+  homepage "http://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner"
+  url "https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-2.8.zip"
+  version "2.8"
+  sha256 "0295365a7e5d4499ec6b46cb6c70f3fa127159b58b73930f675acd0897a6b350"
+  head "https://github.com/SonarSource/sonar-scanner-cli.git"
+
+  bottle :unneeded
+
+  def install
+    # Remove windows files
+    rm_rf Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+    bin.install libexec/"bin/sonar-scanner"
+
+    # sonar-sanner uses this ENV to work out where to install.
+    ENV["SONAR_SCANNER_HOME"] = libexec
+    bin.env_script_all_files libexec/"bin/", :SONAR_SCANNER_HOME => libexec
+  end
+
+  def caveats; <<-EOS.undent
+    If this is your first install, you should adjust the default configuration:
+      #{libexec}/conf/sonar-scanner.properties
```

Comment:
this path used in the script from SONAR_SCANNER_HOME. How can i add link -  etc.install copy script to the /usr/local/etc?